cyclonedx/model/__init__.py

```diff
@@ -1321,7 +1195,8 @@ def __hash__(self) -> int:
     def __repr__(self) -> str:
         return f'<Copyright text={self.text}>'
 
-
+# Importing here to avoid a circular import
+from .tool import Tool  # pylint: disable=wrong-import-position
```

Comment:
I don't know if this is the best solution. We could just put all the Tools stuff into this file, and avoid the circular-imports work-around.

Reply:
I agree since we have a dedicated module(tools.py) to deal with `tool` related stuff we should move it.

Reply:
sure. put all in the `__init__.py`.

Reply:
I spent a few hours trying to solve circular import errors in `cylclonedx.model.bom` (after putting all the tool stuff in `__init__.py` and then gave up when I hit a circular import error in `cyclonedx.serialization`. If I don't stop, I'm going to end up owning all the code . :D I'll leave this as-is for now, and we can do a code restructuring as part of another PR.